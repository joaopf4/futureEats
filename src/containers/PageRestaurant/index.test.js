import React from 'react'
import {
    RestaurantPageWrapper, Title, RestaurantInformation,
    RestaurantImg, RestaurantName, Delivery, Details,
    MenuWrapper, MainMenu, SideDish, Beverage, MenuTitle
} from '../PageRestaurant/style'
import renderer from 'react-test-renderer'

test("Testando styled do RestaurantPageWrapper", () => {
    const newSnapShot = renderer.create(<RestaurantPageWrapper />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do Title", () => {
    const newSnapShot = renderer.create(<Title />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do RestaurantInformation", () => {
    const newSnapShot = renderer.create(<RestaurantInformation />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do RestaurantImg", () => {
    const newSnapShot = renderer.create(<RestaurantImg />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do RestaurantName", () => {
    const newSnapShot = renderer.create(<RestaurantName />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do Delivery", () => {
    const newSnapShot = renderer.create(<Delivery />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do Details", () => {
    const newSnapShot = renderer.create(<Details />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do MenuWrapper", () => {
    const newSnapShot = renderer.create(<MenuWrapper />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do MainMenu", () => {
    const newSnapShot = renderer.create(<MainMenu />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do SideDish", () => {
    const newSnapShot = renderer.create(<SideDish />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do Beverage", () => {
    const newSnapShot = renderer.create(<Beverage />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do MenuTitle", () => {
    const newSnapShot = renderer.create(<MenuTitle />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})