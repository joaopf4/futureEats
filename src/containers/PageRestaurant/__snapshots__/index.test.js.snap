// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando styled do Beverage 1`] = `
<div
  className="sc-iwsKbI eYLkvK"
/>
`;

exports[`Testando styled do Delivery 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-EHOje bCOdrv"
/>
`;

exports[`Testando styled do Details 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-bZQynM lmPunN"
/>
`;

exports[`Testando styled do MainMenu 1`] = `
<div
  className="sc-htoDjs Zziap"
/>
`;

exports[`Testando styled do MenuTitle 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-gZMcBi kjrVwk"
/>
`;

exports[`Testando styled do MenuWrapper 1`] = `
<div
  className="sc-gzVnrw cecYhM"
/>
`;

exports[`Testando styled do RestaurantImg 1`] = `
<img
  className="sc-bxivhb blapSO"
/>
`;

exports[`Testando styled do RestaurantInformation 1`] = `
<div
  className="sc-htpNat eJfEBg"
/>
`;

exports[`Testando styled do RestaurantName 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-ifAKCX fuvCOZ"
/>
`;

exports[`Testando styled do RestaurantPageWrapper 1`] = `
<div
  className="sc-bdVaJa dESAOU"
/>
`;

exports[`Testando styled do SideDish 1`] = `
<div
  className="sc-dnqmqq eAskcY"
/>
`;

exports[`Testando styled do Title 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-bwzfXH hgvgHU"
/>
`;
